posts/workshop-vignette/index.qmd

```diff
@@ -0,0 +1,349 @@
+---
+title: "workshop vignette"
```

Comment:
Could you make this title "What should the first 100 lines of code written during an epidemic look like?" please?

Reply:
done